### ISSUE TITLE
Create new document is not working

### ISSUE THREAD

**Issue by geovannimp** [OfficeDev/office-js#5005]
# Provide required information needed to triage your issue

## Your Environment

* Platform [PC desktop, Mac, iOS, Office on the web]: Office on the web
* Host [Excel, Word, PowerPoint, etc.]: Word
* Office version number: Web
* Operating System: Linux
* Browser (if using Office on the web): Chrome
 
## Expected behavior
Create a new document and write to it content as described in the docs:
https://learn.microsoft.com/pt-br/javascript/api/word/word.application?view=word-js-preview

## Current behavior
`document.body.load("text")` is throwing the exception:
```
{
    "code": "NotAllowed",
    "message": "Não há suporte para a ação Word em um navegador. Verifique OfficeExtension.Error.debugInfo para obter mais informações.",
    "errorLocation": "DocumentCreated.body",
    "statement": "var body = v.body;",
    "surroundingStatements": [
        "var v = context.root._getObjectByReferenceId(\"{d6b1f8db-d91f-4904-8a3a-bc9a1953e855}{7}\");",
        "// >>>>>",
        "var body = v.body;",
        "// <<<<<",
        "// Instantiate {body}",
        "body.load([\"text\"]);"
    ],
    "fullStatements": [
        "Please enable config.extendedErrorLogging to see full statements."
    ]
}
```

## Steps to reproduce
Execute the following script:
```
await Word.run(async (context) => {
    console.log("Opening formatted text in new window");
    const newDocument = context.application.createDocument();
    await context.sync();

    const newDocumentBody = newDocument.body;
    newDocumentBody.load("text");
    await context.sync();
});
```
## Link to live example(s)
Tried to create the live example, but it failed to add the extension.

# Provide additional details
No extra steps

## Context
I was trying to create a new document for the user.

## Useful logs
The logs are in the current behavior section

**Comment by RuizhiSunMS** [comment]
Thank you for reporting this issue, I can repro it. It has been put on our backlog #9484210, the team will investigate and we will reply to you as soon as there is any progress. Thank you for your patience.

**Comment by ElizabethSamuel-MSFT** [comment]
@geovannimp Thanks for reaching out about this. From what I can tell so far, [DocumentCreated.body](https://learn.microsoft.com/pt-br/javascript/api/word/word.documentcreated?view=word-js-preview#word-word-documentcreated-body-member) is currently available in WordApiHiddenDocument 1.3 which doesn't work in Word on the web. See the [Word requirement sets page](https://learn.microsoft.com/pt-br/javascript/api/requirement-sets/word/word-api-requirement-sets?view=word-js-preview) for version and platform support details.

We'll review the samples to ensure this is clear.

@RuizhiSunMS Let's ensure it's noted that customers are looking for support for this feature on the web.

Thanks.

**Comment by ElizabethSamuel-MSFT** [comment]
@geovannimp The issue with the sample should be fixed in Script Lab and the documentation by end of day tomorrow (Dec 17 Pacific time). For details, see the associated PR.
